content/applications/inventory_and_mrp/purchase/products/uom.rst

```diff
@@ -1,6 +1,6 @@
-=================================================
-Purchase in different units of measure than sales
-=================================================
+=========================
+Purchase units of measure
+=========================
 
 When you purchase a product, it may happen that your vendor uses a different unit of measure than
 you do when you sell it. This can cause confusion between sales and purchase representatives. It is
```

Comment:
```suggestion
when it is sold This can cause confusion between sales and purchase representatives. It is
```
please ensure the rest of the doc removes any instance of "you" or "your" -- Thanks

Reply:
Changing this specific instance in this suggestion, but I am not making any other updates to the copy outside of it, per my note at the top of the PR:

**Note:** _This PR was only to update small instructions and replace screenshots. A second, targeted PR for a more robust rewrite will follow once this is merged._

Updates to the copy (which I know are needed) will follow in a separate PR once this is merged.